tests/utils.py

```diff
@@ -20,10 +20,12 @@
 import logging
 import unittest
 
-import airflow.utils.logging
+# import airflow.utils.logging
```

Comment:
Remove rather than comment out please

Reply:
totally unintentional here. added this back in.